scripts/linux-prereqs.sh

```diff
@@ -18,71 +18,58 @@ echo "See https://aka.ms/vsls-docs/linux-prerequisites for manual instructions."
 echo ""
 echo "-------------------------------------------------------------------------------"
 echo ""
-read -p "Contine [Y/n]? " yn
-if [ "$yn" != "y" ] &&  [ "$yn" != "Y" ] && [ "$yn" != "" ] && [ "$yn" != "yes" ] && [ "$yn" != "Yes" ] && [ "$yn" != "YES" ]; then
+read -p "Continue [Y/n]? "
+if ! [[ "$REPLY"  =~ ^[Yy] ]]; then    
```

Comment:
Okay after testing in a few distros it appears to work (though on Ubuntu 16.04 I'll have to tweak the statement in docs to explicitly use bash instead of sh).

The one thing I noticed is here "Y" is the default while now "N" is the default.  So, N should be capitalized in the line above or we need to accept "" as a yes

Reply:
As you prefer, both are really easy changes to the script, By the way I suggest to use bash instead of sh, since is the default interpreter for the script as you can see on line 7

Reply:
Yep, that's the plan.  Lets go with "Y" as the default.

Reply:
If should work as intended now :+1: